src/panel/spoiler_card/spoilerCardActions.js

```diff
@@ -21,24 +21,22 @@ function isValidTitle(index, title) {
 
 function editTitle(index, title, tags) {
 	console.log("editTitle called");
-
-	MainStore.spoilers[index]["title"] = title.trim();
-	MainStore.spoilers[index]["tags"] = commonActions.tagStringToArray(tags);
-
-	console.log("new title on next line");
-	console.log(title);
-
-	console.log("spoiler tags for 1st list on next line");
-	console.log(MainStore.spoilers[0]["tags"]);
+	editList(index, title, tags);
 }
 
 // TODO: disallow duplicate tags?
 function editTags(index, title, tags) {
 	console.log("editTags called");
+	editList(index, title, tags);
+}
 
-	MainStore.spoilers[index]["title"] = title;
+function editList(index, title, tags) {

```

Comment:
What do you think of this change? The methods were almost exactly the same, so extracted them... I'm wondering whether we should just do away with the `editTags` and `editTitle` method though